orchagent/aclorch.cpp

```diff
@@ -4436,6 +4664,71 @@ void AclOrch::doAclTableTask(Consumer &consumer)
                 }
             }
 
+            if (tableTypeName == TABLE_TYPE_UNDERLAY_SET_DSCP || tableTypeName == TABLE_TYPE_UNDERLAY_SET_DSCPV6)
+            {
+                // This table translates into 2 tables.
+                //TABLE_TYPE_MARK_META/V6 on ingress side and TABLE_TYPE_EGR_SET_DSCP on Egress side.
```

Comment:
Is there capability check for creating the new tables?

Reply:
currently no. but i inted to put that in in the next PR as an enhancement.